src/main/java/it/rate/webapp/WebappApplication.java

```diff
@@ -21,12 +37,19 @@ public class WebappApplication implements CommandLineRunner {
   private UserRepository userRepository;
   private VoteRepository voteRepository;
 
+  // check the application.properties file for app.preload-data, default value is true
+  @Value("${app.preload-data:true}")
+  private boolean preloadData;
+
   public static void main(String[] args) {
```

Comment:
I needed to do this because some tests create mocks of repositories which by default don't return anything on .save(), which breaks everything